### PR TITLE
Fix go pbs

### PR DESCRIPTION
**Pull request by shayzluf**
# Description
Fix go pbs script to have the right destination path
**Write why you are making the changes in this pull request**
go pbs script was ditched by most services but is still used by slasher this pr attempt to fix it

### CODE REVIEW

**Comment by shayzluf** [comment]
@terencechain all files weren't updated for vary long time

File: proto/beacon/rpc/v1_gateway/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,16 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```
Comment (nisdas) [inline proto/beacon/rpc/v1_gateway/BUILD.bazel]: why are you adding this whole build file in ?

Comment (shayzluf) [reply proto/beacon/rpc/v1_gateway/BUILD.bazel]: bazel run //:gazelle -- fix made it

Comment (shayzluf) [reply proto/beacon/rpc/v1_gateway/BUILD.bazel]: removed

File: scripts/update-go-pbs.sh [hunk 2]
```diff
@@ -13,13 +13,14 @@ while IFS= read -d $'\0' -r file ; do
...
 arraylength=${#file_list[@]}
-searchstring="prysm/"
+searchstring="prysmaticlabs/prysm/"
 
 # Copy pb.go files from bazel-bin to original folder where .proto is.
 
 for (( i=0; i<${arraylength}; i++ ));
 do
   destination=${file_list[i]#*$searchstring}
+  chmod 755 "$destination"
```
Comment (nisdas) [inline scripts/update-go-pbs.sh]: any reason to change to 755 instead of 644 ?

Comment (shayzluf) [reply scripts/update-go-pbs.sh]: i had 755 and 555 in those directories . i just changed it to the one that worked

Comment (shayzluf) [reply scripts/update-go-pbs.sh]: i guess 644 would suffice. thanks

**Pull request merged**